tests/vmi_configuration_test.go

```diff
@@ -1960,7 +1960,7 @@ var _ = Describe("Configurations", func() {
 				defer expecter.Close()
 
 				res, err := expecter.ExpectBatch([]expect.Batcher{
-					&expect.BSnd{S: "[ $(free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2) -lt 80 ] && echo 'pass' || echo 'fail'\n"},
+					&expect.BSnd{S: "[ $(free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2) -lt 80 ] && echo 'pa''ss'\n"},
 					&expect.BExp{R: "pass"},
 					&expect.BSnd{S: "swapoff -a && dd if=/dev/zero of=/dev/shm/test bs=1k count=118k; echo $?\n"},
```

Comment:
The `swapoff` command has an output that may contain the digit 0 as described in  [issue](https://github.com/kubevirt/kubevirt/issues/3910). We have to make sure the expected 0 is the result of `echo $?` and not of the `swapoff`. This [commit](https://github.com/kubevirt/kubevirt/pull/3882/commits/63b7dab639ce37aff55f2ce6b7b92c67acf9bd58) has a suggestion of how to do it.